admin-portal-frontend/src/app/api/categories.tsx

```diff
@@ -0,0 +1,26 @@
+export interface CategoriesData {
+    title: string;
+    items: [],
+    type: String
+  }
+  
+  const port = process.env.DEV_PORT || 3001;
+  const CATEGORIES_URL = `http://localhost:${port}/api/categories`;
```

Comment:
Make sure to have this url pull from a .env so it works in deployment. Lillian and Edwards pr has a similar file though, maybe you can just use this one once their pr is merged